spec/solution_1_spec.rb

```diff
@@ -0,0 +1,79 @@
+require_relative '../solution_1'
+
+describe Tutorial do
+  let(:klass) { described_class.new }
+
+  describe 'solution' do
+    subject { klass.solution(x) }
+
+    context 'return and reverse number' do
+      let(:x) { 123 }
+      it 'when 123 return 321' do
```

Comment:
Normally, we add empty line for zone (for here variable zone and it block) for easy to review